skywalking/__init__.py

```diff
@@ -40,6 +40,7 @@ class Component(Enum):
     Sanic = 7007
     AioHttp = 7008
     Pyramid = 7009
+    Falcon = 7010
```

Comment:
Can you add this component in the main repo and add a logo in the UI repo?

Reply:
See below.